.github/workflows/build.yaml

```diff
@@ -52,6 +52,8 @@ jobs:
             type=schedule
             type=ref,event=branch
             type=ref,event=pr
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
```

Comment:
```suggestion
            type=pep440,pattern={{version}}
            type=pep440,pattern={{major}}.{{minor}}
```

Reply:
Thanks, this is really helpful. After reading your comment I just copied what we have in https://github.com/openmethane/openmethane/blob/main/.github/workflows/build_docker.yaml#L53C1-L53C44, do you think that's suitable?

Reply:
Yes. It has worked as expected when combined with the CI release process